metadata-ingestion-modules/airflow-plugin/src/datahub_airflow_plugin/client/airflow_generator.py

```diff
@@ -208,6 +209,7 @@ def generate_datajob(
         set_dependencies: bool = True,
         capture_owner: bool = True,
         capture_tags: bool = True,
+        config: DatahubLineageConfig = None,
```

Comment:
I would've preferred to explicitly pass `datajob_url_link: DatajobUrl = DatajobUrl.TASK_INSTANCE` instead of passing the whole config obj

Reply:
just addressed. can you let me know how to trigger the github actions? It seems getting stuck everytime